### PR TITLE
added tests for Domain Range graph

### PR DESCRIPTION
**Pull request by antonsgeorgiev**

### CODE REVIEW

File: test-cypress/integration/explore/class.hierarchy.spec.js [hunk 1]
```diff
@@ -143,6 +143,42 @@ describe('Class hierarchy screen validation', () => {
...
+        let className = ':Region';
+        searchForClass(className);
+        cy.wait(500);
+        cy.get('.btn-primary').click();
+        cy.wait(500);
+        getDomainRangeGraphHeader().should('contain','Domain-Range graph');
+        getLegendContainer().should('be.visible');
+        getLegendContainer().should('contain', 'main class node');
+        getLegendContainer().should('contain', 'class node');
+        getLegendContainer().should('contain', 'collapsed property');
```
Comment (svilenvelikov) [inline test-cypress/integration/explore/class.hierarchy.spec.js]: I think you can chain the above like:
```
getLegendContainer().should('contain', 'main class node')
.and('contain', 'class node')
.and('contain', 'collapsed property');
```
Same applies for the checks below. The idea is to make the selection once and not to repeat it again and again without need.

File: test-cypress/integration/explore/class.hierarchy.spec.js [hunk 2]
```diff
@@ -191,4 +227,7 @@ describe('Class hierarchy screen validation', () => {
         cy.get('.modal-footer .confirm-btn').click();
         cy.get('.modal').should('not.be.visible');
     }
+
```
Comment (svilenvelikov) [inline test-cypress/integration/explore/class.hierarchy.spec.js]: You left many consecutive blank lines. Can you collapse them to a single blank line where necessary please.

File: test-cypress/integration/explore/class.hierarchy.spec.js [hunk 3]
```diff
@@ -143,6 +143,42 @@ describe('Class hierarchy screen validation', () => {
             });
     });
 
+    it('Test domain range graph', () => {
+        let className = ':Region';
+        searchForClass(className);
+        cy.wait(500);
```
[the remark this answers is not in the document]
Comment (antonsgeorgiev) [reply test-cypress/integration/explore/class.hierarchy.spec.js]: Removed unnecessary waits in commit beb405691242fb594436842557e670b96b7e6a8b

File: test-cypress/integration/explore/class.hierarchy.spec.js [hunk 4]
```diff
@@ -143,6 +143,42 @@ describe('Class hierarchy screen validation', () => {
...
+        cy.get('.btn-primary').click();
```
[the remark this answers is not in the document]
Comment (antonsgeorgiev) [reply test-cypress/integration/explore/class.hierarchy.spec.js]: Added unique class selector and a separate getter in commit beb405691242fb594436842557e670b96b7e6a8b

File: test-cypress/integration/explore/class.hierarchy.spec.js (hunk 1 above)
Comment (antonsgeorgiev) [reply test-cypress/integration/explore/class.hierarchy.spec.js]: Chained as shown in commit beb405691242fb594436842557e670b96b7e6a8b

File: test-cypress/integration/explore/class.hierarchy.spec.js (hunk 2 above)
Comment (antonsgeorgiev) [reply test-cypress/integration/explore/class.hierarchy.spec.js]: Removed blanks in commit beb405691242fb594436842557e670b96b7e6a8b

**Pull request merged**